nahusenayElias/Team_4
# DRD-23: Modify .gitignore so that files are included in repo

## Related ticket
[DRD-23](https://edu-team4-react24k.atlassian.net/browse/DRD-23?atlOrigin=eyJpIjoiZTI2ODhhYWI4NzJkNGNlYTkxZjg0N2ZiMzYxNzA3ZDAiLCJwIjoiaiJ9)

## In this PR:
- Modified .gitignore so that folders containing images are included in the repo (database contains only references/paths to images, not the images themselves)

## Testing instructions:
- checkout branch
- run lando start
- run lando drush cr
- check that the drupal backend "About page" and "About us" react page are displaying an image
